database/migrations/2020_05_29_095139_create_survey_responses_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateSurveyResponsesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('survey_responses', function (Blueprint $table) {
+            $table->bigIncrements('id');
+            $table->string('response_id');
+            $table->unsignedBigInteger('question_id');
```

Comment:
you can merge this like so
`$table->foreignId('question_id')->constrained()->onDelete('cascade');`
you would then do away with the code on line 22.

you could take a look at this (https://laravel.com/docs/7.x/migrations#foreign-key-constraints) for more info

Reply:
Noted.